apps/web/components/search/search-dialog.tsx

```diff
@@ -208,214 +223,268 @@ export function SearchDialog({ open, onOpenChange }: SearchDialogProps) {
+											<h3 className="text-lg font-semibold">
+												Found {searchResults.length} result{searchResults.length !== 1 ? "s" : ""}
+											</h3>
+											{selectedFiles.size > 0 && (
+												<div className="flex items-center gap-2">
+													<span className="text-sm text-muted-foreground">{selectedFiles.size} selected</span>
+													<Button size="sm" onClick={() => addTagToSelected("important")}>
+														<Tag className="h-3 w-3 mr-1" />
+														Add Tag
+													</Button>
 												</div>
-											</CardContent>
-										</Card>
-									))}
-								</div>
-							</div>
-						</div>
-					) : query && query.trim() ? (
-						<div className="flex-1 flex items-center justify-center text-center py-12">
-							<div className="text-muted-foreground">
-								<Search className="h-12 w-12 mx-auto mb-4 opacity-50" />
-								<p className="text-lg mb-2">No results found</p>
-								<p className="text-sm">Try different keywords or check your spelling</p>
-							</div>
-						</div>
-					) : (
-						<div className="flex-1 flex items-center justify-center text-center py-12">
-							<div className="text-muted-foreground">
-								<Search className="h-12 w-12 mx-auto mb-4 opacity-50" />
-								<p className="text-lg mb-2">Start searching</p>
-								<p className="text-sm">Enter keywords to find your files or try the examples above</p>
+											)}
+										</div>
+
+										<div className="flex-1 overflow-y-auto py-3">
+											<div className="grid gap-3">
+												{searchResults.map(file => (
+													<Card
+														key={file.id}
+														className={`cursor-pointer transition-colors hover:bg-accent/50 ${
+															selectedFiles.has(file.id) ? "ring-2 ring-blue-500 bg-blue-50/50" : ""
+														}`}
+														onClick={() => toggleFileSelection(file.id)}
+													>
+														<CardContent className="p-4">
+															<div className="flex items-start justify-between">
+																<div className="flex items-center gap-3 flex-1">
+																	{getFileIcon(file.type)}
+																	<div className="flex-1 min-w-0">
+																		<h4 className="font-medium truncate">{file.name}</h4>
+																		<div className="flex items-center gap-2 text-sm text-muted-foreground">
+																			<span>{file.modified}</span>
+																			<span>•</span>
+																			<span>{file.size}</span>
+																			<span>•</span>
+																			<span className="capitalize">{file.type}</span>
+																		</div>
+																	</div>
+																</div>
+																<div className="flex flex-wrap gap-1 ml-4">
+																	{file.tags.map(tag => (
+																		<Badge
+																			key={tag.id}
+																			variant="secondary"
+																			className={`text-xs ${tag.color} text-white`}
```

Comment:
```suggestion
																			className={`text-xs ${tag.color} text-white hover:bg-neutral-100 dark:hover:bg-neutral-900 hover:text-black hover:border-neutral-400 dark:hover:text-neutral-200 dark:hover:border-neutral-600`}
```
Hover state was a little weird in light mode, so I added this as a temp fix. In the future, well have a constants file with the tag color defaults, light and dark variants and their hover colors